src/main/webapp/script.js

```diff
@@ -146,6 +146,11 @@ function addAuthenticationButton(authenticationUrl, buttonStyle, buttonText, nav
   authenticationButtonNavbar.appendChild(authenticationButtonItem);
 }
 
+function loadSignup() {
+  isUserRegistered();
+  fetchMentorExperience();
+}
+
 /**
  * Redirect user in signup page to index if they are already registered.
```

Comment:
Use simple present tense verbs for JavaDoc.

Reply:
Done!